operators/understanding/olm-packaging-format.adoc

```diff
@@ -17,12 +17,14 @@ include::modules/olm-about-opm.adoc[leveloffset=+2]
 * See xref:../../cli_reference/opm-cli.adoc#opm-cli[CLI tools] for steps on installing the `opm` CLI.
 
 include::modules/olm-package-manifest-format.adoc[leveloffset=+1]
+.Additional resources
```

Comment:
This also can be changed to `== Additional resources` with the `[id="additional-resources_"]` anchor.

Reply:
Same as above.